script/ablist.yml

```diff
@@ -12,6 +12,7 @@
 - type: simple
   source: http://ncdj.org/style-guide/
   note: If possible, describe exacly what this is.
+  # note: Assumes/implies that a person with a disability is deficient or inferior to others. When possible, specify the functional ability or its restriction.
```

Comment:
@wuz any reason this is commented out? It looks good!

Reply:
Yep! It makes the test fail :/ 

It's the test I reference in `2.` in [this issue](https://github.com/retextjs/retext-equality/issues/60#issue-373619261) fails. Here is the code that is related to that test: https://github.com/retextjs/retext-equality/blob/master/test.js#L14

Reply:
Ahh okay, sorry I missed that! Feel free to uncomment this line and change the test, no problem!

Reply:
Okie doke! Thanks! I'll update this soon! 😄